### DISCUSSION TITLE
從衛星地圖影像辨識非洲動物生態分布

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS4719-C/Python-Community-GS4719-C#141] [General]
2019的文章了，可以參考一下：https://www.facebook.com/RiChiTech/posts/2065188203528487/

**Comment by enoch-lightning** [comment]
連大象都看不出來，那我就更看不出來了。

**Comment by xCk27x** [comment]
以一個成年男性肩寬約45cm為例，在這張照片中大概只會佔1~2個像素而已。若解析度能高到辨識出人類的話，我猜需要高至5*5cm左右，感覺還需要很長一段時間才能達到。
